addons/stock_account/tests/test_stockvaluation.py

```diff
@@ -2364,7 +2364,7 @@ def test_average_perpetual_8(self):
 
         self.assertAlmostEqual(self.product1.standard_price, 10.0)
 
-    def test_average_perpetual_8(self):
+    def test_average_perpetual_9(self):
```

Comment:
@Whenrow I renamed this one as it prevents the previous test to execute by shadowing it.
I hope it's ok ?